pom.xml

```diff
@@ -893,9 +893,12 @@
                                         <mysql.80.image>registry.redhat.io/rhel8/mysql-80</mysql.80.image>
                                         <mariadb.103.image>registry.redhat.io/rhel8/mariadb-103</mariadb.103.image>
                                         <mariadb.105.image>registry.redhat.io/rhel8/mariadb-105</mariadb.105.image>
+                                        <mariadb.10.image>registry.redhat.io/rhel8/mariadb-1011</mariadb.10.image>
```

Comment:
On one hand, it's your decision what you use and I won't block this change. But on the other hand, I have hard time you really need this. You only test OpenShift, not baremetal, and I didn't find any test that is using this image. I suppose I just missed the one? Which OpenShift test is failing because of this image, please?

Reply:
@michalvavrik Right, we only want to tests in the Openshift mode. Now that I understand what `-Dtest=!OpenShiftTodoDemoIT` does I suspect it was set on a run when we hit the mariadb.10.image image pull from docker. I can remove it from the change.

Reply:
It's small thing, but yeah, please remove it, I'll let formatting validation check pass and merge this.